cmd/snap/cmd_help.go

```diff
@@ -39,7 +39,7 @@ The help command displays information about snap commands.
 
 // addHelp adds --help like what go-flags would do for us, but hidden
 func addHelp(parser *flags.Parser) error {
-	var help struct {
```

Comment:
Note that this got added just to ensure the check works and it will be removed before this goes out of "draft"